scripts/create.js

```diff
@@ -0,0 +1,102 @@
+import { getElement } from './utility.js'
+
+/* CREATE CARD FORM */
+export const createCardForm = getElement('createCardForm')
```

Comment:
do you also export const ? I thougt only functions, to use them in other files...